bundle/src/main/java/com/adobe/acs/commons/wcm/impl/SiteMapServlet.java

```diff
@@ -298,8 +305,25 @@ private void write(Page page, XMLStreamWriter stream, ResourceResolver resolver)
         stream.writeEndElement();
     }
 
-    private boolean isHidden(final Page page) {
-        return page.getProperties().get(this.excludeFromSiteMapProperty, false);
+    private boolean isHiddenOrProtected(Page page) {
```

Comment:
Dont follow this method naming? ..  what does `...OrProtected(..)` mean?

It seems the more semnatic names for these 2 methods would be:

`isHiddenByPageProperty(..)` and  `isHiddenByPageTemplate(...)`

Reply:
there might be pages like article which will be completely available to read only when user logs in, so these kind of pages can be considered as protected pages, but agree by what you suggested thanks